build/makelib/common.mk

```diff
@@ -74,7 +74,7 @@ endif
 # set the version number. you should not need to do this
 # for the majority of scenarios.
 ifeq ($(origin VERSION), undefined)
-VERSION := $(shell git describe --dirty --always --tags)
+VERSION := $(shell git describe --dirty --always --tags | sed 's/-g/./g')
```

Comment:
what i was really trying to get to is what would the builds look like up on https://release.rook.io/master/ ?

would the intermediate builds, in between official builds, look like v0.5.0-12? so basically everything from the `-g` on is truncated?